modules/ROOT/pages/dataweave-types.adoc

```diff
@@ -472,17 +472,17 @@ Note that like the Java class, the iterator is designed to be consumed only once
 [[dw_type_null]]
 == Null (dw::Core Type)
 
-DataWeave defines a Null data type for `null` values. It is important to note that `null` 
+DataWeave defines a Null data type for `null` values. It is important to note that `null`
 is the value, while Null is the type. DataWeave functions do not recognize `Null` as
-a `null` value. 
+a `null` value.
 
 You can determine whether a DataWeave function accepts `null` values by
 looking in the DataWeave reference documentation for a function signature
 that accepts a Null type. For example, the
 xref:dw-core-functions-flatten.adoc#flatten2[flatten] function has two function
 signatures, one of which is `flatten(Null): Null`. The signature indicates that
-`flatten` can accept `null` values. The xref:dw-core-functions-plusplus.adoc[++] 
-(concatenate) function, which lacks a signature for handling value so the Null 
+`flatten` can accept `null` values. The xref:dw-core-functions-plusplus.adoc[++]
+(concatenate) function, which lacks a signature for handling value so the Null
 type, does not accept `null`.
```

Comment:
The second sentence is not grammatical:

The function, which lacks...so the Null type, does not accept `null`?

Please clarify.